x-pack/legacy/plugins/apm/server/lib/transactions/get_transaction/index.ts

```diff
@@ -18,12 +18,16 @@ import {
 } from '../../helpers/setup_request';
 import { ProcessorEvent } from '../../../../common/processor_event';
 
-export async function getTransaction(
-  transactionId: string,
-  traceId: string,
-  setup: Setup & SetupTimeRange & SetupUIFilters
-) {
-  const { start, end, uiFiltersES, client, indices } = setup;
+export async function getTransaction({
+  transactionId,
+  traceId,
+  setup
+}: {
+  transactionId: string;
+  traceId: string;
+  setup: Setup & SetupTimeRange & SetupUIFilters;
+}) {
```

Comment:
I converted the function signature to use a destructured object. Therefore the noise.